SS-common/flash/SS_s25fl_tests.c

```diff
@@ -0,0 +1,186 @@
+/*
+ * test_s25fl.c
+ *
+ *  Created on: Feb 23, 2020
+ *      Author: Mikolaj Wielgus
+ */
+
+#include "unity_fixture.h"
+#include "SS_s25fl.h"
+
+TEST_GROUP(s25fl);
+
+TEST_GROUP_RUNNER(s25fl)
+{
+	RUN_TEST_CASE(s25fl, rems_id);
+	//RUN_TEST_CASE(s25fl, erase_all);
+	RUN_TEST_CASE(s25fl, erase);
+	RUN_TEST_CASE(s25fl, wait);
+	//RUN_TEST_CASE(s25fl, wait_dma);
+	RUN_TEST_CASE(s25fl, write_read_bytes);
+	RUN_TEST_CASE(s25fl, write_read_bytes_dma);
+	RUN_TEST_CASE(s25fl, write_read_page);
+	RUN_TEST_CASE(s25fl, write_read_page_dma);
+}
+
+TEST_SETUP(s25fl)
+{
+	SS_s25fl_init();
```

Comment:
Test setup is called before each test and allocates memory for semaphore multiple times